src/router.js

```diff
@@ -1,25 +1,49 @@
-const express = require('express');
-const jwt = require('jsonwebtoken');
+const express = require("express");
+const jwt = require("jsonwebtoken");
+const secret = process.env.JWT_SECRET
+console.log(secret)
 
 const router = express.Router();
 
 const mockUser = {
-    username: 'authguy',
-    password: 'mypassword',
-    profile: {
-        firstName: 'Chris',
-        lastName: 'Wolstenholme',
-        age: 43
-    }
+  username: "authguy",
+  password: "mypassword",
+  profile: {
+    firstName: "Chris",
+    lastName: "Wolstenholme",
+    age: 43,
+  },
 };
 
-router.post('/login', (req, res) => {
+router.post("/login", (req, res) => {
+  const { username, password } = req.body;
 
-});
+  if (username === mockUser.username && password === mockUser.password) {
+    const token = jwt.sign(mockUser.username, secret);
 
-router.get('/profile', (req, res) => {
-  
+    res.json({
+      token,
+    });
+  } else {
+    res.json({
```

Comment:
You need to specify a status here, otherwise it looks like the request is successful